docs/sphinx/examples/cpp/other/builder/vqe_h2_builder.cpp

```diff
@@ -25,7 +27,7 @@ namespace cudaq {
 // Define a function that applies a general SO(4) rotation to
 // the builder on the provided qubits with the provided parameters.
 // Note we keep this qubit and parameter arguments as auto as these
-// will default to taking the qubits and variational parameters (QuakeValues)
+// will default to taking the qubits and variational parameters (`QuakeValue`s)
```

Comment:
This isn't a doxygen comment so the backticks don't do anything here. In doxygen, they are the equivalent to inserting `<code> ... </code>` afaict.

Reply:
The spell checker is configured to ignore anything marked as code (marked by backticks). The spell checker checks for spelling mistakes in comments since it seemed a reasonable option - we use line comments for doc comments as well, and we use normal comments in samples that we should spell check. Plus, I think it is easier to just write all comments similar to doc comments to build the habit, rather than omitting the ticks in comment but not in doc comments. 

In short, the backticks are added so the spell checker does not complain about QuakeValue (which refers to code and hence indicated as such).